tdigest/merging_digest.go

```diff
@@ -355,9 +355,6 @@ func (td *MergingDigest) Merge(other *MergingDigest) {
 	}
 }
 
-var _ gob.GobEncoder = &MergingDigest{}
-var _ gob.GobDecoder = &MergingDigest{}
-
```

Comment:
any reason these lines are being removed? this is a compile-time assertion that MergingDigest satisfies the GobEncoder/GobDecoder interfaces, which is still relevant since gob is still being used (at least, as of this PR)